### PR TITLE
Update foreman_ansible to 1.4.4 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [x] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Thanks! Depends on https://github.com/theforeman/foreman-packaging/pull/1525

### CODE REVIEW

**Comment by mmoll** [comment]
This needs asset precompilation added like in #1513 - the "assets" directory should be "foreman_ansible" now.

**Comment by dLobatog** [comment]
Updated with all steps needed for assets precompilation, thanks for the tip @mmoll

File: plugins/ruby-foreman-ansible/debian/control [hunk 1]
```diff
@@ -3,15 +3,15 @@ Section: ruby
...
   foreman-sqlite3 (>= 1.12.0~rc1)
 Standards-Version: 3.9.3
 Homepage: https://github.com/theforeman/foreman_ansible
 
 Package: ruby-foreman-ansible
 Architecture: all
 Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0~rc1),
-  ruby-foreman-deface (<<2.0.0), ruby-foreman-tasks (>= 0.8.2-2),
-  ruby-foreman-tasks (<<0.9.0)
+  ruby-foreman-deface (<<2.0.0), foreman-assets(>= 1.12.0~rc1),
```
Comment (mmoll) [inline plugins/ruby-foreman-ansible/debian/control]: foreman-assets is only a buildtime dependency

**Comment by mmoll** [comment]
seems to work just fine, otherwise

**Comment by dLobatog** [comment]
@mmoll Updated the PR to remove -assets as a runtime dep, thanks for the review

**Comment by mmoll** [comment]
merged, gracias @dLobatog!

**Pull request merged**